css/style.css

```diff
@@ -1,9 +1,101 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+#root {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+#root > * {
+  width: 60%;
+}
+header #logo {
+  max-height: 2em;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+#header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0.5em 0;
+  font-size: 1.5em;
+  color: grey;
+}
+
+#header-container nav a:hover {
+  color: #ff6915;
+}
+
+#karma-container div {
+  display: flex;
+  font-size: 1.5rem;
+}
+.karma-icon {
+  max-width: 7em;
+}
+
+.karma-tile {
+  text-align: center;
+}
+
+#hero-container,
+#karma-container,
+#footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#hero-container {
+  min-height: 65vh;
+  line-height: 0.5;
+  background-image: url("../img/first-background.jpg");
```

Comment:
i have a suggestion for background properties here:
```css
#hero-container {
    background-size: 100%;
    background-position: center;
    background-repeat: no-repeat;
}
```
`min-height:65vh` looks a bit much to me. 
we could probably reduce to down to 50vh.